src/vs/workbench/browser/actions/developerActions.ts

```diff
@@ -139,17 +139,29 @@ class ToggleScreencastModeAction extends Action {
 		}));
 
 		const keyboardMarker = append(container, $('.screencast-keyboard'));
+
 		disposables.add(toDisposable(() => keyboardMarker.remove()));
 
+		const updateKeyboardFontSize = () => {
+			keyboardMarker.style.fontSize = `${this.configurationService.getValue<number>('screencastMode.fontSize')}px`;
+			keyboardMarker.style.height = keyboardMarker.style.lineHeight = `${this.configurationService.getValue<number>('screencastMode.fontSize') / 0.56}px`;
```

Comment:
This can very likely be avoided and set in CSS.

Reply:
Fixed